flare_portal/experiments/templates/experiments/participant_bulk_delete_form.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block title %}Delete Participant - {{ experiment.name }}{% endblock title %}
```

Comment:
Nitpick: would be slightly nicer as an exact match of the title below,

```suggestion
{% block title %}Delete {{ participants|length }} Participant{{ participants|length|pluralize }} - {{ experiment.name }}{% endblock title %}
```